plugins/inventory/cobbler.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Orion Poplawski <orion@nwra.com>
+# Copyright (c) 2020 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: cobbler
+    plugin_type: inventory
+    short_description: cobbler inventory source
+    description:
+        - Get inventory hosts from the cobbler service.
+        - "Uses a configuration file as an inventory source, it must end in ``.cobbler.yml`` or ``.cobbler.yaml`` and has a ``plugin: cobbler`` entry."
+    extends_documentation_fragment:
+        - inventory_cache
+    options:
+      plugin:
+        description: the name of this plugin, it should always be set to 'cobbler' for this plugin to recognize it as it's own.
+        required: True
+        choices: ['cobbler']
+      url:
+        description: url to cobbler
+        default: 'http://cobbler/cobbler_api'
+        env:
+            - name: COBBLER_SERVER
+      user:
+        description: cobbler authentication user
+        required: False
+        env:
+            - name: COBBLER_USER
+      password:
+        description: cobbler authentication password
+        required: False
+        env:
+            - name: COBBLER_PASSWORD
+      exclude_profiles:
+        description: profiles to exclude from inventory
+        type: list
+        default: []
+      group_by:
+        description: keys to group hosts by
+        type: list
+        default: [ 'mgmt_classes', 'owners', 'status' ]
+      group:
+        description: group to place all hosts into
+        default: cobbler
+      group_prefix:
+        description: prefix to apply to cobbler groups
+        default: cobbler_
+      want_facts:
+        description: Toggle, if True the plugin will retrieve host facts from the server
```

Comment:
```suggestion
        description: Toggle, if C(true) the plugin will retrieve host facts from the server
```
`C(...)` is used for values (https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation).